frontend/src/components/GameView/Timer/Timer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Line } from "rc-progress";
+import PropTypes from "prop-types";
+import timer from "./Timer.scss";
+
+export default function Timer({ currentTime, maxTime }) {
+  const progress = Math.floor((currentTime / maxTime) * 100);
+  const minutes = Math.floor(currentTime / 60);
+  let seconds = currentTime - minutes * 60;
+
+  let textColor = timer.white;
+  let strokeColor = timer.base;
+  const red = "#cc0000";
+
+  if (seconds < 60) {
+    seconds = `0${seconds}`;
```

Comment:
I think you meant to check if `second < 10` not `60`? Currently it would show seconds as `059` I believe.

Reply:
Yes ofc, I've changed it to 10.